tests/integration/e2e/SchedulePage.test.tsx

```diff
@@ -0,0 +1,199 @@
+        const testParentCourse = testMeeting.courseInstance.course;
+        const testChildCourse = childMeetings[0].courseInstance.course;
+        await renderResult.findAllByText(testChildCourse.title);
+
+        // Before setting the child course's "Same As" value to the parent,
+        // we are filtering the Course Admin table by area, because there are
+        // courses with the same title and differing catalog numbers. When
+        // selecting the edit button, we want to make sure we are selecting
+        // the intended one.
+        const areaLabelText = 'The table will be filtered as selected in this area dropdown filter';
+        const areaFilter = renderResult
+          .getByLabelText(areaLabelText) as HTMLSelectElement;
+        fireEvent.change(areaFilter,
+          { target: { value: testChildCourse.area.name } });
+        const catalogText = 'The table will be filtered as characters are typed in this course filter field';
+        const catalogFilter = renderResult
+          .getByLabelText(catalogText) as HTMLInputElement;
+        fireEvent.change(catalogFilter,
+          { target: { value: `${testChildCourse.prefix} ${testChildCourse.number}` } });
+        // Open the child course modal to edit the "Same As" value
+        const editButton = await renderResult
+          .findByLabelText(`Edit course information for ${testChildCourse.title}`,
+            { exact: false });
+        fireEvent.click(editButton);
+        await renderResult.findByRole('dialog');
+        const sameAsSelect = renderResult.getByLabelText('Same As', { exact: false }) as HTMLSelectElement;
+        fireEvent.change(
+          sameAsSelect,
+          { target: { value: testParentCourse.id } }
+        );
+        const submitButton = renderResult.getByText('Submit');
+        fireEvent.click(submitButton);
+        await wait(() => !renderResult.queryByText('required fields', { exact: false }));
+
+        // Navigate to the schedule page and wait for the table to render
+        const scheduleTab = await renderResult.findByText('Schedule');
+        fireEvent.click(scheduleTab);
+        const parentCatalogNumber = `${testParentCourse.prefix} ${testParentCourse.number}`;
+        await renderResult.findAllByText(parentCatalogNumber, { exact: false });
+
+        // Make sure the child course displays the parent course's meeting information
+        const parentCourseDay = dayEnumToString(testMeeting.day);
+        const parentStartTime = PGTime.toDisplay(testMeeting.startTime);
+        const parentEndTime = PGTime.toDisplay(testMeeting.endTime);
+        const parentLocation = `${testMeeting.room.building.name} ${testMeeting.room.name}`;
+        const parentMeetingInfo = `${parentCourseDay}, ${parentStartTime} to ${parentEndTime} in ${parentLocation}`;
+        const childCourseInfo = `${testChildCourse.prefix} ${testChildCourse.number} on ${parentMeetingInfo}`;
+        await renderResult.findByText(childCourseInfo);
```

Comment:
Maybe not the best place to tag this, but I am still seeing the issue where the popover appears for both the child and the parent, with one typically overlapping the other. Looking back through the commit history, I think you were on the right track using a reference to the prefix/number of the clicked course, though we could potentially simplify the setup a bit by just prepending the catalog number to the `instanceId` when we call `setCurrentPopver`, or compare with the state value.